tekton-task-images/helm/Dockerfile

```
@@ -2,14 +2,25 @@ FROM registry.access.redhat.com/ubi8/ubi-minimal:8.3
 
 USER root
 
+ENV YQ_VERSION=4.11.2
+
 RUN microdnf install --assumeyes --nodocs openssl tar gzip && \
     microdnf update && \
     microdnf clean all
 
 ADD VERSION /tmp/version
+# helm
 RUN source /tmp/version && \
-  curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 && \
-  chmod 700 get_helm.sh && \
-  ./get_helm.sh --version ${HELM_VERSION}
+    curl -skL -o /tmp/helm.tar.gz https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz && \
+    tar -C /tmp -xzf /tmp/helm.tar.gz && \
+    mv -v /tmp/linux-amd64/helm /usr/local/bin && \
+    chmod -R 775 /usr/local/bin/helm && \
+    rm -rf /tmp/linux-amd64 && \
```

We should not use `curl -k` as this turns off TLS/SSL verification. We can also shorten this a bit since `helm` already has `0755` permissions.
```suggestion
    curl -sL https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz | \
    tar zxf - -C /usr/local/bin --strip-components 1 linux-amd64/helm && \
```